Content.Shared/GameObjects/Components/Damage/SharedDamageableComponent.cs

```diff
@@ -20,9 +20,7 @@ namespace Content.Shared.GameObjects.Components.Damage
     ///     Component that allows attached entities to take damage.
     ///     This basic version never dies (thus can take an indefinite amount of damage).
     /// </summary>
-    [RegisterComponent]
-    [ComponentReference(typeof(IDamageableComponent))]
-    public class DamageableComponent : Component, IDamageableComponent, IRadiationAct
+    public abstract class SharedDamageableComponent : Component, IDamageableComponent, IRadiationAct
```

Comment:
Don't forget to restore the RegisterComponent and ComponentReference if you do follow my suggestion to delete the client / server specific subclasses.

Reply:
Restored in https://github.com/space-wizards/space-station-14/pull/2783/commits/c49fec0341bce74946a9b670c8a12468299f5081